src/modules/Builder/components/Condition/Condition.styles.ts

```diff
@@ -0,0 +1,79 @@
+import { styled } from '@mui/material';
+
+import { StyledFlexTopCenter, variables, theme } from 'shared/styles';
+import { SelectController, InputController } from 'shared/components/FormComponents';
+
+export const StyledCondition = styled(StyledFlexTopCenter)`
+  gap: 0.4rem;
+  padding: ${theme.spacing(0.8, 1.2)};
+  border-radius: ${variables.borderRadius.xs};
+  position: relative;
+
+  & > * {
```

Comment:
is it possible to use more specific rules?

Reply:
done